tikv-client/src/main/java/com/pingcap/tikv/TiConfiguration.java

```diff
@@ -116,6 +117,9 @@ public class TiConfiguration implements Serializable {
   private long connRecycleTime = getTimeAsSeconds(DEF_TIKV_CONN_RECYCLE_TIME);
   private long certReloadInterval = getTimeAsSeconds(DEF_TIKV_TLS_RELOAD_INTERVAL);
 
+  private String networkMappingName = "";
```

Comment:
Why add two private variables here and never access to them.

Reply:
This is a special configuration for some users. Maybe we can ignore it here.